build.gradle

```diff
@@ -7,15 +7,21 @@ buildscript {
         springBootVersion = '2.4.5'
         blackDuckCommonVersion = '59.0.4'
         junitPlatformCustomTestTags = 'docker, battery'
+        javaTargetCompatibility = 8
+        javaSourceCompatibility = 8
     }
 
     apply from: 'https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-repositories.gradle', to: buildscript
-    apply from: 'https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-dependencies.gradle', to: buildscript
+
     dependencies {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
     }
 }
 
+plugins {
```

Comment:
You don't need this plugin applied. I should remove it from the example file. I like the task graph that this plugin gives, so I was using it in the test/sample project.